doc/03-Configuration.md

```diff
@@ -0,0 +1,120 @@
+# Configuration
+
+The configuration for Icinga Notifications is twofold.
+The main configuration resides in the database,
+shared between the Icinga Notifications daemon and Icinga Notifications Web.
+However, as the Icinga Notifications daemon needs to know how to access this database and some further settings,
+it needs its own configuration file as well.
+
+This configuration is stored in `/etc/icinga-notifications/config.yml`.
+See [config.example.yml](../config.example.yml) for an example configuration.
+
+## Top Level Configuration
+
+### HTTP API Configuration
+
+The HTTP API listener can be used both for submission and for debugging purposes.
+
+| Option         | Description                                                          |
+|----------------|----------------------------------------------------------------------|
+| listen         | Address to bind to, port included. (Example: `localhost:5680`)       |
+| debug-password | Password expected via HTTP Basic Authentication for debug endpoints. |
+
+### Icinga Web 2
+
+The `icingaweb2-url` is expected to point to the base directory of your Icinga Web 2 installation,
+i.e., `https://example.com/icingaweb2/`, to be used for URL creation.
+
+### Channels Directory
+
+All available Icinga Notification Channels should reside in the `channels-dir` directory.
+For a package installation, the default will point to the correct location and must not be changed.
```

Comment:
I'd write it as "Icinga Notifications channels", i.e. one less upper-case letter.